src/decorations.tsx

```diff
@@ -0,0 +1,61 @@
+interface KawaiiDecorationsProps {
+  color: string
+}
+
+export function KawaiiDecorations({ color }: KawaiiDecorationsProps) {
```

Comment:
I think the decoration layer is a good idea! I also believe that we’ll see more types of decorations from the community. Therefore, I would prefer to use the same name as this theme (`KawaiiCatDecoration`), so we can add more cute decorations in the future without naming conflict.

Reply:
makes sense, i fixed the naming!